aip/general/0193.md

```diff
@@ -75,132 +86,191 @@ following:
 }
 ```
 
-### Details
 
-Google defines a set of [standard detail payloads][details] for error details,
-which cover most common needs for API errors. Services **should** use these
-standard details payloads when feasible.
+#### google.rpc.Status
 
-Structured details with machine-readable identifiers **must** be used if users
-will need to write code against a specific aspect of the error. Error message
-strings **may** change over time; however, if an error message does not have a
-machine-readable identifier _in addition to_ the `code` field, changing the
-error message **must** be considered a backwards-incompatible change.
+Services must return a
+[`google.rpc.Status`][Status] message when an API error occurs. For each
```

Comment:
It looks like these two lines could be reformatted. The first line can take more before max width. This will need to cascade through the paragraph. Your editor should be able to assist.

Reply:
I reformatted the entire file to meet max width.